test/benchmark/bench_linux_and_aster.sh

```diff
@@ -10,34 +10,52 @@ command -v jq >/dev/null 2>&1 || { echo >&2 "jq is not installed. Aborting."; ex
 
 # Script directory
 BENCHMARK_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
-# Kernel image 
-KERNEL_DIR="/root/dependency"
-LINUX_KERNEL="${KERNEL_DIR}/vmlinuz"
-KERNEL_VERSION="5.15.0-105-generic"
-MODULES_DIR="${BENCHMARK_DIR}/../build/initramfs/lib/modules/${KERNEL_VERSION}/kernel"
+# Dependencies for Linux
+LINUX_DEPENDENCIES_DIR="/opt/linux_dependencies"
```

Comment:
First of all, the Dockefile also downloads files to this directory. So you need to change the directory name in both locations.

https://github.com/asterinas/asterinas/blob/6fcad6ce143b8f65e8be76297eb95a9cb56da22e/tools/docker/Dockerfile.jinja#L26-L29

Second, the name "linux_dependencies" is vague. It is unclear whether it means files that Linux depends on or Linux files that Asterinas depends on. I suggest calling it "linux_binary_cache" because the directory is intended to be used as a cache for downloaded Linux binaries, including the kernel, modules, and vdso.

In addition, `test/Makefile` should have use the cached files in this directory:

https://github.com/asterinas/asterinas/blob/6fcad6ce143b8f65e8be76297eb95a9cb56da22e/test/Makefile#L64-L66

Could you also fix it so that it won't download vdso again to a temporary build directory?

Reply:
Certainly. `linux_binary_cache` is a better description. And it should also be applied to the module [repository](https://github.com/asterinas/linux_binary_cache/tree/main). 

I will also add the kernel modules used by `iperf3` in #1331.